blockchain_api/artifacts/src/chaincode/opetbot/opetbot.go

```diff
@@ -24,6 +24,10 @@ type User struct {
     Documents []string `json:"documents"`
 }
 
+/*
+    The struct is used to store undefined json tree of the user's document
```

Comment:
s/undefined/arbitrary
(replace undefined with arbitrary)

Reply:
Or even better `Document represents an arbitrary json document with user data`.

See https://blog.golang.org/godoc-documenting-go-code, the recommended way is to use code names (like `Document`) inside the descriptions.